tools/sm_teledump.lua

```diff
@@ -73,12 +70,16 @@ local MEMTRACK = { -- {{{
     { 0x0A68, 0x2, 'Flash suit' },
     { 0x0A76, 0x2, 'Hyper beam' },
     { 0x0AF6, 0x2, 'Samus X' },
+    { 0x0AF8, 0x2, 'Samus subpixel X' },
     { 0x0AFA, 0x2, 'Samus Y' },
+    { 0x0AFC, 0x2, 'Samus subpixel Y' },
```

Comment:
I want to add subpixel support, so might as well have this here for the future.  It's not a big deal for most presets though; I'll probably have it default to 0000 or FFFF most of the time.  Currently it appears subpixels are set to 0000 anyway so it'll take a bit of work to support it... although interestingly if Samus' pose is running left or right then subpixels might be 3000 or D000; I guess the game gives Samus one frame of movement in that case.